### PR TITLE
Add a Cypress test to open a listing page (and remove unused test files)

### LINKED ISSUE #480: individual listing view doesn't load (what's left here: add a test)
[REMEMBER: a bug should be understandable years later by someone who will never talk to the reporter or assignee.]

STEPS TO REPRODUCE:
1. `cd backend/core && yarn db:reseed && cd -`
2. `yarn dev:all`
3. go to `/listings` and open a single listing

EXPECTED RESULTS:
The listing loads

OBSERVED RESULTS:
I get a Page Not Found with the backend saying:
```
 BACKEND_CORE [Nest] 3886912   - 08/26/2021, 1:02:01 PM   [ExceptionsHandler] missing FROM-clause entry for table "unitssummary" +41ms
 BACKEND_CORE QueryFailedError: missing FROM-clause entry for table "unitssummary"
 BACKEND_CORE     at new QueryFailedError (/usr/local/google/home/wrl/git-repo/bloom-detroit/node_modules/typeorm/error/QueryFailedError.js:12:28)
 BACKEND_CORE     at PostgresQueryRunner.<anonymous> (/usr/local/google/home/wrl/git-repo/bloom-detroit/node_modules/typeorm/driver/postgres/PostgresQueryRunner.js:248:31)
 BACKEND_CORE     at step (/usr/local/google/home/wrl/git-repo/bloom-detroit/node_modules/typeorm/node_modules/tslib/tslib.js:143:27)
 BACKEND_CORE     at Object.throw (/usr/local/google/home/wrl/git-repo/bloom-detroit/node_modules/typeorm/node_modules/tslib/tslib.js:124:57)
 BACKEND_CORE     at rejected (/usr/local/google/home/wrl/git-repo/bloom-detroit/node_modules/typeorm/node_modules/tslib/tslib.js:115:69)
 BACKEND_CORE     at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

ADDITIONAL INFORMATION:

### PR DESCRIPTION
**Pull request by anders-schneider**
## Issue

- Closes #480

## Description

This change adds a test that verifies that we can open a listing detail page without error. It also removes a few unused test files and refactors the `package.json` files in `sites/public` and at the top level.

Note: I tried to make it so that this test created its own listing as a "before" step. It took too long to figure out the permissions, so I gave up on that in favor of just assuming that there already exists at least one listing already in the database. This is a fair assumption, because the yarn interface defined in `sites/public/package.json` makes it so that the database is reseeded (with the seed listings) before the test is run.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Prototype/POC (not to merge)
- [X] This change is a refactor/address technical debt
- [ ] This change requires a documentation update
- [ ] This change requires a SQL Script
- [X] Test addition

## How Can This Be Tested/Reviewed?

`$ yarn test:app:public:headless`

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes
- [ ] I have run `yarn generate:client` if I made backend changes

### CODE REVIEW

**Comment by anders-schneider** [comment]
Sorry @willrlin I'm looking into the test failure and have made this a draft for now (not ready to review). I'll convert it to a real PR and add a note here when it's ready for review.

**Comment by anders-schneider** [comment]
@willrlin Okay this is ready for review now. I tried to get this working in a better way (with the test creating the listing that it relies on, instead of relying on db:reseed to create a listing), but I've sunk more time into this than I should've. I still think this is valuable (now we have a test that actually opens the individual listing page), but I don't think it warrants more time.

**Review by willrlin — APPROVED** [review]

File: sites/public/cypress/fixtures/listing.json [hunk 1]
```diff
@@ -1,1271 +0,0 @@
-{
```
[the remark this answers is not in the document]
Comment (anders-schneider) [reply sites/public/cypress/fixtures/listing.json:1]: I didn't dig into the history, but I confirmed that the current Cypress tests don't rely on it.

File: sites/public/cypress/support/index.d.ts [hunk 2]
```diff
@@ -1,20 +0,0 @@
-declare namespace Cypress {
```
[the remark this answers is not in the document]
Comment (anders-schneider) [reply sites/public/cypress/support/index.d.ts:1]: Also not sure what the history is, but we don't need it anymore.

**Pull request merged**